src/task_run_diagnostic_reports.py

```diff
@@ -85,20 +88,44 @@ def create_ticket(group, disease, touchstone,
         description = get_version_url(report, version, config) if \
             report_success else \
             "Auto-run failed with error: {}".format(error)
-        issue = yt.create_issue(Project(vimc_project_id),
-                                summary.format(group, disease, touchstone),
-                                description)
-        yt.run_command(
-            Command([issue],
-                    "for {} implementer {} tag {} tag {} tag {}".format(
-                        report.assignee,
-                        report.assignee,
-                        group, disease,
-                        touchstone)))
+        create_tags(yt, group, disease, touchstone, report)
+        query = "tag: {} AND tag: {} AND tag: {} AND tag: {} " \
+                "AND state: Incoming"
+        existing_issues = yt.get_issues(
+            query.format(disease, touchstone, group, report.name),
+            ["id", "summary", "description", "tags(name)"])
+        summary = summary.format(group, disease, touchstone)
+        if len(existing_issues) > 0:
+            existing_issue = existing_issues[0]
+            yt.update_issue(existing_issue, summary, description)
```

Comment:
Will this not overwrite the description rather than appending the new report version? I thought we had decided we would append the new report version to the existing description, since we want to retain the previous report version as well (as it is probably not a superseded version, but rather a different scenario to the one currently being dealt with). 
It might be really nice to include the scenario alonside the link to each report version - we do get this as a task parameter.

Reply:
Yes this overwrites, our final resolution was that YouTrack's "view history" was sufficient for seeing previous descriptions! Good idea about adding scenario, might be helpful for understanding the context, will add that.

Reply:
Ah right, I'd forgotten that. Thanks!